test/src/HexGrid.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import HexGrid from '../../src/HexGrid';
+
+test('HexGrid should render correctly with default props', () => {
+  const tree = renderer.create(
+    <HexGrid>
+      <div>child</div>
```

Comment:
<div> is not allowed in SVG structure but for testing purposes I guess it doesn't matter?